### ISSUE TITLE
Populate MCU List from CAN Bus

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#51]
## User story
1. As a user
2. I want to see a list of attached ECUs in the GUI
3. So that I can choose the one I want to flash / reset / etc.

## Acceptance criteria
* MCU ID and Name are collected from CAN Bus via UDS and shown in the GUI
* MCU can be selected to perform further actions

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
